src/autopas/containers/verletClusterLists/VerletClusterListsRebuilder.h

```diff
@@ -301,66 +311,72 @@ class VerletClusterListsRebuilder {
   }
 
   /**
-   * Decides if for a given tower and a neighbor tower, clusters of the tower should contain clusters of the neighbor
-   * tower as neighbors if newton 3 is enabled.
+   * Decides if a given neighbor tower is a forward neighbor to a given tower.
+   * A forward neighbor is either in a interaction cell with a higher index
+   * or in the same interaction cell with a higher tower index.
    *
-   * Works in a way to help the VerletClustersColoringTraversal have no data races.
+   * Helps the VerletClustersColoringTraversal to have no data races.
    *
-   * @param towerIndexX The x-coordinate of the given tower.
-   * @param towerIndexY The y-coordinate of the given tower.
-   * @param neighborIndexX The x-coordinate of the given neighbor tower.
-   * @param neighborIndexY The y-coordinate of the given neighbor tower.
+   * @param towerIndexX The x-index of the given tower.
+   * @param towerIndexY The y-index of the given tower.
+   * @param neighborIndexX The x-index of the given neighbor tower.
+   * @param neighborIndexY The y-index of the given neighbor tower.
    * @return True, if clusters of the given tower should contain clusters of the given neighbor tower as neighbors with
```

Comment:
documentation of return should be adapted.

Reply:
6d823227